<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GriddingProcess</class>
 <widget class="QWidget" name="GriddingProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gridding Process</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QDialogButtonBox" name="yesNo_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>700</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This button is used to confirm or deny the CL confirmation question. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="CL_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>660</width>
        <height>371</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is your command line output for the gbtgridder program.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;See the y/n buttons below to confirm or deny the confirmation question. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>700</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Utopia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Command Line Output</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="CL_error">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>660</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the command line output for errors.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(111, 111, 111);
alternate-background-color: rgb(170, 0, 0)</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="plainText">
       <string>stderr ouput (if needed)
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>30</y>
        <width>700</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Utopia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your gridding has begun! </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CL_output</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>yesNo_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CL_error</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>CL_error</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
